components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -10,9 +10,8 @@ ul {
 }
 
 app-page-picker {
-  margin-left: auto;
-  margin-right: auto;
-  width: 225px;
+  padding-left: 35px;
+  padding-bottom: 35px;
```

Comment:
This is fixing the margins in the page-picker inside the services-bar, before it was not well aligned.